HostWormsGame.ps1

```diff
@@ -32,7 +32,13 @@ function Update-Options() {
 }
 
 function Start-Worms() {
-	Start-Process (Join-Path $installDirPath "WA.exe") "/host"
+    # See http://worms2d.info/Command-line_options
+    #
+    # The scheme parameter is documented here:
+    # http://worms2d.info/WormNET_(Worms_Armageddon)#Channels
+    #
+    # 'We' means 4 worms per team (don't ask)
+	& $wa wa:host?scheme=We
```

Comment:
My previous understanding was that if you used these parameters, it gave you a link which everyone had to join using and wouldn't sync settings (hence the clients get out of sync) - it seems like they can also join the game normally and hence probably won't get out of sync. I merged it since it seems likely to work, and we'll change it if not.